presto-main/src/main/java/com/facebook/presto/sql/analyzer/AggregationAnalyzer.java

```diff
@@ -134,6 +144,13 @@ public void analyze(Expression expression)
     private class Visitor
             extends AstVisitor<Boolean, Void>
     {
+        private final Set<Expression> columnReferences;
```

Comment:
Since this is a non-static inner class, you don't need this field. You can just access `columnReferences` in the parent class